geometry_msgs/Vector3 noise_vector
geometry_msgs/Vector3 beta_vector
---
bool success
